Makefile

```
@@ -79,13 +79,16 @@ GINKGO_FOCUS ?=
 GINKGO_SKIP ?=
 GINKGO_NODES ?= 1
 GINKGO_TIMEOUT ?= 3h
-E2E_CONF_FILE ?= $(abspath test/e2e/config/vsphere-dev.yaml)
+E2E_CONF_FILE ?= $(abspath test/e2e/config/vsphere.yaml)
+E2E_CAPV_MODE ?= legacy
```

```suggestion
E2E_CAPV_MODE ?= govmomi
```
(same in other places)